example.py

```diff
@@ -96,7 +96,7 @@ def __init__(self):
         lo.addWidget(iconwidget)
         lo.addWidget(QtWidgets.QLabel('IconWidget'))
         iconwidgetholder.setLayout(lo)
-        iconwidget2 = qta.IconWidget('mdi.web', color='blue')
+        iconwidget2 = qta.IconWidget('mdi.web', color='blue', size=QtCore.QSize(24, 24))
```

Comment:
Since this changes the size, I think we will need to update the [examples screenshot/gif](https://github.com/spyder-ide/qtawesome/blob/master/qtawesome-screenshot.gif). Other option is to change this to pass the default value (16) so in that case there will be no need to update the screenshot.

Also, we need to update the corresponding text in the examples entry in the [`README.md`](https://github.com/spyder-ide/qtawesome/tree/master#examples) and the docs [`usage.rst`](https://github.com/spyder-ide/qtawesome/blob/master/docs/source/usage.rst) to follow this change

Reply:
Thanks! I haven't thought of the two other files.

As a side note, why is `IconWidget` not mentioned in [the docs](https://qtawesome.readthedocs.io/en/latest/index.html)?

Reply:
That's a good point @StSav012 ! Maybe we need to add some other params to the `automodule` directive or calling other directive in the [API Reference page](https://github.com/spyder-ide/qtawesome/blob/cd4768d3e7f5eb9a2ab7b639dfe866060a3dec63/docs/source/api_documentation.rst) so the `IconWidget` class docs appear? Let us know if you would like to help us checking that!

Reply:
I've tried to add `IconWidget` line before https://github.com/spyder-ide/qtawesome/blob/0bda7bf65c842b23711169d6563683b021300e13/qtawesome/__init__.py#L13 (or where's the best place for that?). The item appears in the docs. But `sphinx` is more clever than me, and it adds ugly empty lines about every single method of `QLabel`. As much as I'd like to tackle the problem, I have neither enough experience nor enough time these days. Maybe, closer to the weekend.

Reply:
> or where's the best place for that?

Maybe we need to check if adding a `Custom widgets` section could be worthy but as a first approach makes sense to me :+1:

> As much as I'd like to tackle the problem, I have neither enough experience nor enough time these days. Maybe, closer to the weekend.

No worries, thank you for your time checking how the directive works and how we could improve the docs!